src/commands/git-metadata/__tests__/gitdb.test.ts

```diff
@@ -588,7 +863,7 @@ describe('gitdb', () => {
         },
       ],
       revparse: [{input: '--is-shallow-repository', output: 'false'}],
-      version: [],
+      version: [{input: undefined, output: newGitVersion}],
```

Comment:
ℹ️ this is changed because I've changed the way we unshallow: we first check version. Let me know if you disagree with the change